.github/workflows/release-21_extrinsic-ordering-check-from-two.yml

```diff
@@ -0,0 +1,95 @@
+# This workflow performs the Extrinsic Ordering Check on demand using two reference binaries
+
+name: Release - Extrinsic API Check with reference bins
+on:
+  workflow_dispatch:
+    inputs:
+      reference_binary_url:
+        description: A url to a Linux binary for the node containing the reference runtime to test against
+        default: https://releases.parity.io/polkadot/x86_64-debian:stretch/v0.9.26/polkadot
+        required: true
+      binary_url:
+        description: A url to a Linux binary for the node containing the runtime to test
+        default: https://releases.parity.io/polkadot/x86_64-debian:stretch/v0.9.27-rc1/polkadot
+        required: true
+
+jobs:
+  check:
+    name: Run check
+    runs-on: ubuntu-latest
+    env:
+      BIN_URL: ${{github.event.inputs.binary_url}}
+      REF_URL: ${{github.event.inputs.reference_binary_url}}
+    strategy:
+      matrix:
+        chain: [polkadot, kusama, westend, rococo]
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Fetch reference binary
+        run: |
+          echo Fetching $REF_URL
+          curl $REF_URL -o polkadot-ref
+          chmod a+x polkadot-ref
+          ./polkadot-ref --version
+
+      - name: Fetch test binary
+        run: |
+          echo Fetching $BIN_URL
+          curl $BIN_URL -o polkadot
+          chmod a+x polkadot
+          ./polkadot --version
+
+      - name: Start local reference node
+        run: |
+          echo Running reference on ${{ matrix.chain }}
+          ./polkadot-ref --chain=${{ matrix.chain }}-local --rpc-port=9934 --ws-port=9945 --base-path=polkadot-ref-base/ &
+
+      - name: Start local test node
+        run: |
+          echo Running test on ${{ matrix.chain }}
+          ./polkadot --chain=${{ matrix.chain }}-local &
+
+      - name: Prepare output
+        run: |
+          VERSION=$(./polkadot --version)
+          echo "Metadata comparison:" >> output.txt
+          echo "Date: $(date)" >> output.txt
+          echo "Reference: $REF_URL" >> output.txt
+          echo "Test binary: $BIN_URL" >> output.txt
+          echo "Target version: $VERSION" >> output.txt
+          echo "Local chain: ${{ matrix.chain }}" >> output.txt
```

Comment:
```suggestion
          echo "Local chain: ${{ matrix.chain }}-local" >> output.txt
```

Reply:
it already uses the terminology "Local chain" explicitly for this